ideascube/blog/views.py

```diff
@@ -64,6 +64,7 @@ class ContentCreate(CreateView):
     form_class = ContentForm
 
     def get_initial(self):
+        self.object = Content(author = self.request.user)
```

Comment:
Ho, you right.

The thing here is that the form have no `author` field anymore and so the `initial['author']` is not taken into account. So I need a bound form to be able to provide a "default" value for a model field not in the form.

I will try to hide the field instead of remove it.